### PR TITLE
Quadrat: Fix styling of post meta

### LINKED ISSUE #4606: Quadrat: Add padding to Post Meta
Currently, Post Meta is too close to the Post Title which makes the layout feel more constrained.

Adding 50px padding to the post meta area feels reasonable and more comfortable.

Captura de ecrã 2021-09-15, às 11 16 43

| Current      | Intended |
| ----------- | ----------- |
| Captura de ecrã 2021-09-15, às 11 17 15  |  Captura de ecrã 2021-09-15, às 11 16 52  |

### PR DESCRIPTION
**Pull request by danieldudzic**
Closes #4606

### CODE REVIEW

File: quadrat/sass/templates/_meta.scss [hunk 1]
```diff
@@ -1,37 +1,61 @@
...
-		// Needed while https://github.com/WordPress/gutenberg/issues/31710 is sorted.
-		color: transparent;
-		a {
-			color: var(--wp--custom--color--foreground);
-			&:not(:last-child):after {
-				// to be removed after https://github.com/WordPress/gutenberg/pull/32812 lands
-				content: ",";
-			}
+	.wp-block-post-author__content {
+		align-items: center;
```
Comment (mikachan) [inline quadrat/sass/templates/_meta.scss]: I don't think we need `align-items: center` here, it looks like `justify-content` is enough. It also seems to make the author content horizontally centered:

Screenshot 2021-09-23 at 11 10 37

**Review by mikachan — COMMENTED** [review]
This is working well for me! Only spotted the one issue with the author alignment.

I think this also fixes https://github.com/Automattic/themes/issues/4605

File: quadrat/sass/templates/_meta.scss (hunk 1 above)
Comment (danieldudzic) [reply quadrat/sass/templates/_meta.scss]: Thanks, done!

**Review by mikachan — APPROVED** [review]
LGTM!

**Comment by scruffian** [comment]
Deployed in D67367-code

**Pull request merged**